source/billing_system.html.md.erb

```diff
@@ -0,0 +1,43 @@
+
+- 2 applications written in Golang: the `paas-billing-collector` and the `paas-billing-api`, both sharing the [paas-billing code base](https://github.com/alphagov/paas-billing)
+- the GOV.UK PaaS billing Postgres database application, which uses stored functions to calculate tenants' bills in the database
+
+## Accessing GOV.UK PaaS billing
+
+To access GOV.UK PaaS billing information, use the PaaS Admin web application through the *Explore your costs and usage* link on your organisation’s page. You can also access more detailed information, such as resource IDs and charging plan details, through the [PaaS Billing API](https://github.com/alphagov/paas-billing/README.md).
+
+PaaS Admin calls the `\billable_events` endpoint of the API. The API then queries the billing database to get the required data and calls a stored function. For example, [the `get_tenant_bill` stored function](https://github.com/alphagov/paas-billing/blob/main/billing-db/sprocs/get_tenant_bill.sql)) is the database entry point for calculating tenant bills.
+
+
+## How GOV.UK PaaS billing works
+
+We use database SQL code to calculate tenants' bills. There are 2 parts to the procedure for calculating bills:
+
+1. Assembling what needs to be billed
+
+    We put together a set of billable resources and services and a time frame across which to bill them. The [`get_tenant_bill_api`](https://github.com/alphagov/paas-billing/blob/main/billing-db/sprocs/get_tenant_bill_api.sql) stored function calculates bills and is invoked through the API. Information is collated from the event-sourced Cloud Foundry usage events for the queried time frame and written to a temporary database table by a stored function.
+
+    This stored function takes an organisation name as well as the start and end date of the queried time frame as arguments. It then goes through the *resources* table for the tenant and time frame, collects records of which services or resources the tenants used, and writes them to a temporary table.
+
+2. Calculating the bill
+
+    The [`calculate_bill`](https://github.com/alphagov/paas-billing/blob/main/billing-db/sprocs/calculate_bill.sql) stored function calculates the bill. This function accepts no parameters. It takes the contents of the temporary table populated by `get_tenant_bill_api` and calculates the bill, taking account of factors such as VAT rates and currency exchange rates. The exchange rates are important because many services provisioned by GOV.UK PaaS are billed in US dollars (USD), but we produce bills for our tenants in British pounds (GBP).
+
+
+### How we calculate bills
+
+The PaaS billing system calculates tenants’ costs, based on pricing information collected from our service providers. We do this by writing simple formulae in the billing database, and each service or resource we provide to tenants has a formula to calculate the cost.
+
+
+```
+($storage_in_mb/1024) * ($time_in_seconds/2678401) * $external_price
```

Comment:
If it's a different formula for every resource, is it worth saying that this is an example?
If it's the same, then maybe clarify that this is it

Also, I would finder clearer if this was an equation, e.g.:
```suggestion
Total charge = ($storage_in_mb/1024) * ($time_in_seconds/2678401) * $external_price
```